docs/lang/branch.md

```diff
@@ -125,9 +125,9 @@ switch (i) {
 }
 ```
 
-以上代码运行后输出的结果为 WIKI，因为 break 的存在，接下来的语句就不会继续被执行了。default 语句不需要 break，因为下面没有语句了。
+The output of above code is WIKI, because the following statements will not continue to be executed since the break statement exists. The default statement does not need a break, because there is no statement below.
```

Comment:
输出加 mono 吧
```suggestion
The output of above code is WIKI, because the following statements will not continue to be executed for the 
 existence of break statement. The default statement does not need a break, because there is no statement below.
```